src/python/pants/init/engine_initializer.py

```diff
@@ -399,6 +401,7 @@ def build_root_singleton() -> BuildRoot:
     rules = (
       [
         RootRule(Console),
+        RootRule(GlobalOptionValueContainer),
```

Comment:
Er: see my comment above. This doesn't/shouldn't need to be a RootRule. It can be computed from the OptionsBootstrapper, and so can be a plain `@rule` declared in the file I linked.